### ISSUE TITLE
"Send Me Reset Password Instructions" causes error

### ISSUE THREAD

**Issue by karenmajewicz** [BTAA-Geospatial-Data-Project/geomg#59]
Filled in email address, clicked button, and received a generic error

**Comment by ewlarson** [comment]
@mberkowski // Can I assign you this ticket? Very low priority.

Not sure how you want email to run on the GEOMG server, but this is just a configuration issue. I imagine we might have more mail coming from the application as reporting tools and bulk action processes mature.

**Comment by mberkowski** [comment]
Basic outbound mail should already work. If we need to tune the appearance of outbound addresses we may have to do more.

**Comment by mberkowski** [comment]
Basic configuration should work with:

```
  config.action_mailer.delivery_method = :sendmail
  config.action_mailer.perform_deliveries = true
  config.action_mailer.raise_delivery_errors = true
  config.action_mailer.default_options = {from: %{"B1G Geoportal" <swadm@#{`hostname`.strip}>}}
  config.action_mailer.default_url_options = { :host => `hostname`.strip }
```

**Comment by mberkowski** [comment]
Fixed in 617ca0ed870b06563f9d87acac71359812db2e4f
